src/txgraph.cpp

```diff
@@ -3451,20 +3451,6 @@ TxGraph::Ref::~Ref()
     }
 }
 
-TxGraph::Ref& TxGraph::Ref::operator=(Ref&& other) noexcept
-{
-    // Unlink the current graph, if any.
-    if (m_graph) m_graph->UnlinkRef(m_index);
-    // Inform the other's graph about the move, if any.
-    if (other.m_graph) other.m_graph->UpdateRef(other.m_index, *this);
```

Comment:
note: also hinted in https://github.com/bitcoin/bitcoin/pull/33157#discussion_r2423058928